scripts/docker/aws-sdk.Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+
+FROM centos:centos7.8.2003
+COPY dss-ansible/artifacts/dss-gcc510-*.rpm ./
+RUN set -eux \
+	&& yum install -y \
+        epel-release &&  \
+    yum install -y \
```

Comment:
this line is not required along with preceding &&

Reply:
Marking resolved per other comment